erpnext_com/www/docs/user/manual/en/human-resources/holiday-list.md

```diff
@@ -3,39 +3,72 @@
 
-Most organizations have a standard Holiday-List for their employees. Some even have different holiday lists based on the different locations or departments.
+Most organizations have a standard Holiday List for their employees. Some even have different holiday lists based on the different Locations or Departments.
 
-To add a new Holiday List, go to:
+To access Holiday List, go to:
 
-`Human Resources (HR) > Leave and Holiday > Holiday List`
+> Home > Human Resources (HR) > Leaves > Holiday List
 
-Click on New to add new Holiday List.
 
-### New Holiday List
 
-Give a name to Holiday List. It can be based in Fiscal Year or location or department as application. Also select From and To Date for the Holiday List.
+## 1. How to create a Holiday List
+
+To create a new Holiday List, go to:
+
+1. Holiday List > New.
+2. Give a name to the Holiday List. It can be based on the Fiscal Year or Location or Department as per the requirement. 
+3. Select From Date and To Date for the Holiday List.
+
 
 <img class="screenshot" alt="Holiday List" src="{{docs_base_url}}/assets/img/human-resources/holiday-list-1.png">
 
-You can quickly add Weekly Off in the Holiday List as following.
+## 2. Features 
+
+Some of the additional features in the Holiday List doctype are as follows:
+
+### 1. Add Weekly Holidays 
+
+You can quickly add Weekly Offs in the Holiday List as following:
+
+1. In the 'Add Weekly Holidays' section, select the day in the Weekly Off field.
+2. Click on 'Add to Holidays' button.
+3. Save.
+
+Once the Weekly Off is added, it is reflected in the Holidays table. 
+
+
+> Note: You can add multiple days in the Weekly Offs.
 
 <img class="screenshot" alt="Holiday List" src="{{docs_base_url}}/assets/img/human-resources/holiday-list-2.gif">
 
-After that, you can also add specific days (like festival holidays) manually.
+
+After that, you can also add specific days (like festival holidays) manually by clicking on the 'Add row' option in the Holidays table.
 
 <img class="screenshot" alt="Holiday List" src="{{docs_base_url}}/assets/img/human-resources/holiday-list-3.png">
 
+> Note: 
+Each time a new holiday is updated in the Holidays table, the Total Holidays field gets updated.
+
+
+### 3. Holiday List in Employee 
+
+If you have created multiple Holiday List, select a specific Holiday List for an Employee in the respective master.
+
+When an Employee applies for Leave, the days mentioned in the Holiday List will not be counted, as they are holidays already. 
+
+
 
-### Holiday List in Employee
+> Note: If you have specified a Holiday List in the Employee master, then that Holiday List will be given more priority as compared to the default Holiday List of the company.
+You can form as many holiday lists as you wish. For example, if you have a factory, you can have one list for the factory workers and another list for office staff. You can manage between many lists by linking a Holiday List to the respective Employee.
```

Comment:
```suggestion
> You can form as many holiday lists as you wish. For example, if you have a factory, you can have one list for the factory workers and another list for office staff. You can manage between many lists by linking a Holiday List to the respective Employee.
```
take care how the formatting looks

Reply:
Even if you put > , it takes it as one block only.

Reply:
the point is to have a line break, not one big chunk of text, so line break, then >\